PULL_REQUEST_TEMPLATE

```diff
@@ -1,3 +1,6 @@
+##### ABBREVIATED REVIEW BETWEEN 2.13.X AND MASTER IS IN EFFECT
```

Comment:
I'm confused. I don't think it should be added in 2.13.x by default, because by default the abbreviated review should happen on the master branch, not the 2.13.x branch. https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines#PullRequestGuidelines-NotesonReviewProcess

Reply:
Hmm. You make a valid point. And now I don't think I know which of the following is appropriate:
- Note on 2.13.x only
- Note on master only
- Note on both 2.13.x and master

I'm not sure which usage gets us the best outcome.

Reply:
I honestly kind of feel like it makes more sense this way (backporting instead of forwardporting). But perhaps, we should just have folks copy it themselves?

Reply:
The DDF guideline is to forward port, not back port: https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines.